sig-api-machinery/annual-report-2022.md

```diff
@@ -106,16 +109,16 @@ Include any other ways you measure group membership
 
 Operational tasks in [sig-governance.md]:
 
-- [ ] [README.md] reviewed for accuracy and updated if needed
+- [x] [README.md] reviewed for accuracy and updated if needed
 - [ ] [CONTRIBUTING.md] reviewed for accuracy and updated if needed
       (or created if missing and your contributor steps and experience are different or more
       in-depth than the documentation listed in the general [contributor guide] and [devel] folder.)
 - [ ] Subprojects list and linked OWNERS files in [sigs.yaml] reviewed for accuracy and updated if needed
```

Comment:
@fedebongio - the only invalid URL in `sigs.yaml` for API Machinery is:
https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/master/OWNERS 

That dir does not exist anymore. Should we remove this?

Reply:
it moved to here: https://github.com/kubernetes/kubernetes/blob/master/pkg/controlplane/OWNERS